ansible_collections/arista/avd/roles/eos_designs/templates/designs/l3ls-evpn/tenant_evpn_vxlan/leaf-tenant-router-bgp-evpn-vrfs.j2

```diff
@@ -46,8 +49,16 @@
 {%                     if tenants[tenant].vrfs[vrf].bgp_peers[bgp_peer].nodes is arista.avd.defined %}
 {%                         if inventory_hostname in tenants[tenant].vrfs[vrf].bgp_peers[bgp_peer].nodes %}
 {%                             set neighbor_params = tenants[tenant].vrfs[vrf].bgp_peers[bgp_peer] %}
-{%                             if neighbor_params.address_family is not arista.avd.defined %}
-{%                                 do neighbor_params.update({'address_family': ['ipv4']}) %}
+{%                             if neighbor_params.address_family is arista.avd.defined %}
```

Comment:
@ClausHolbechArista @titom73 @onurgashi @emilotep - I refactored keeping the current behavior in eos_designs, but I believe we can further abstract this by not requiring the address_family to be defined at all and using ansible.netcommon.ipv4/ipv6 to detect it automatically -> https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters_ipaddr.html

What do you think?

Reply:
Looks good to have AF autodetection for v4/v6

Reply:
Yes, looks good to me.

Reply:
implemented this approach